secure_message/common/utilities.py

```diff
@@ -70,56 +70,76 @@ def process_paginated_list(paginated_list, host_url, user, message_args, endpoin
     return messages, links
 
 
-def get_business_details_by_ru(rus):
-    """Function to retrieve business details from ru using the party service"""
+def get_from_details(messages):
+    """looks up the details for the from users"""
+    update_internal_messages_from(messages)
+    from_details = party.get_user_details(get_messages_from_external(messages))
+    external_messages = [x for x in messages if x['from_internal'] is False]
```

Comment:
'x' doesn't seem like a very explicit name maybe rename this to something more meaningful and makes sense in context

Reply:
I disagree @MIslam92 . It s a pythonic comprehension to do it . Are you refereing to the function name ?

Reply:
So much has changed, everyone reviewing this should ignore all the comments that came before it